supporting_scripts/course-scripts/quick-course-setup/config.ini

```
@@ -0,0 +1,38 @@
+[Settings]
+students = 1000
+commits = 0
+exercises = 0
```

I think this default might be more useful? I do not have a strong opinion, does not need to be changed for me :) 

```suggestion
students = 2000
commits = 2
exercises = 1
```